src/components/TodoItem.tsx

```diff
@@ -0,0 +1,59 @@
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo;
+  onRemove: (userId: number) => void;
+  isDeleting: boolean;
+  todosTransform: number[];
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  onRemove,
+  isDeleting,
+  todosTransform,
+}) => {
+  const { completed, title, id } = todo;
+
+  return (
+    <div
+      className={classNames('todo', {
+        completed: completed,
+      })}
+    >
+      <label className="todo__status-label">
+        <input
+          type="checkbox"
+          className="todo__status"
+        />
+      </label>
+
+      <span className="todo__title">{title}</span>
+
+      <button
+        type="button"
+        className="todo__remove"
+        onClick={()=> {
+          onRemove(id);
+        }}
```

Comment:
Create separate handler instead of anonymous function

Make it for all similar cases